packages/dds/tree/src/test/simple-tree/treeApi.spec.ts

```diff
@@ -830,18 +833,21 @@ describe("treeNodeApi", () => {
 			assert.deepEqual(eventLog, [new Set(["key1", "key2", "key3"])]);
 		});
 
-		it(`'nodeChanged' does not include the names of changed properties (arrayNode)`, () => {
+		it(`'nodeChanged' does not have changed properties (arrayNode)`, () => {
 			const sb = new SchemaFactory("test");
 			class TestNode extends sb.array("root", [sb.number]) {}
 
 			const view = getView(new TreeViewConfiguration({ schema: TestNode }));
 			view.initialize([1, 2]);
 			const root = view.root;
 
-			const eventLog: ReadonlySet<string>[] = [];
-			Tree.on(root, "nodeChanged", ({ changedProperties }) =>
-				eventLog.push(changedProperties),
-			);
+			let eventFireCount = 0;
+			// @ts-expect-error The Tree.on() overload for nodeChanged event for an array node should not have a payload.
+			// It still exists, but it should be an empty set for now.
```

Comment:
This test maybe just wants to disappear instead... But then I don't know how to maintain confidence that we're not exposing the payload by mistake on array nodes.